DataConnectors/Cloudflare/Cloudflare_API_FunctionApp.json

```diff
@@ -0,0 +1,130 @@
+{
+    "id": "CloudflareDataConnector",
+    "title": "Box",
```

Comment:
Title Name should PROVIDER NAME APPLIANCE NAME

Reply:
"Box" was left there accidentally. Fixed.